Runtime/Native/Android/NativeClient.cs

```diff
@@ -361,9 +346,9 @@ private bool InitializeJavaCrashHandler(String minidumpUrl, String databasePath,
                 AndroidJNI.NewStringUTF(minidumpUrl),
                 AndroidJNI.NewStringUTF(databasePath),
                 AndroidJNI.NewStringUTF(_crashHandlerPath),
-                AndroidJNIHelper.ConvertToJNIArray(new string[0]),
-                AndroidJNIHelper.ConvertToJNIArray(new string[0]),
-                AndroidJNIHelper.ConvertToJNIArray(attachments.ToArray()),
+                AndroidJNIHelper.ConvertToJNIArray(Array.Empty<string>()),
```

Comment:
I think this is also the newest .net syntax - can you bring back my old version?

Array.Empty is a sugar syntax on top of something we already had.  it's just better to be compatible with more apis